docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -994,6 +994,26 @@ You can use the [`collapse-rename-collapsedheight` codemod](https://github.com/m
 
   You can use the [`use-transitionprops` codemod](https://github.com/mui-org/material-ui/tree/HEAD/packages/material-ui-codemod#use-transitionprops) for automatic migration.
 
+- Default `anchorOrigin.vertical` is changed
+
+  ```diff
+  {
+    - vertical: 'bottom',
+    + vertical: 'top',
+  }
+  ```
```

Comment:
I see two issues with this new entry. 1. We don't explain why we did the breaking change 2. We don't make the migration path easy for the developers (the diff is about what happened, which is interesting but not how they can revert back).

Reply:
Prefer something like this (active forme):

````suggestion
- Change the default value of `anchorOrigin.vertical` to follow the Material Design guidelines. The menu now displays below the anchor instead of over it. You can restore the previous behavior with:

  ```diff
   <Menu
  +  anchorOrigin={{
  +    vertical: 'top',
  +    horizontal: 'left',
  +  }}
  ```
````

Reply:
a lot better 👍